libcudacxx/include/cuda/__functional/address_stability.h

```diff
@@ -48,6 +48,13 @@ _CCCL_INLINE_VAR constexpr bool proclaims_copyable_arguments_v = proclaims_copya
 template <typename F>
 struct __callable_permitting_copied_arguments : F
 {
+#if _CCCL_STD_VER < 2017
+  template <typename G>
+  _LIBCUDACXX_HIDE_FROM_ABI constexpr __callable_permitting_copied_arguments(G&& g)
+      : F(::cuda::std::forward<G>(g))
+  {}
+#endif
```

Comment:
Below C++17, `__callable_permitting_copied_arguments` is not an aggregate so it needs a ctor.